### ISSUE TITLE
Link of repository is the old one

### ISSUE THREAD

**Issue by DrFabach** [medkit-lib/medkit#11]
In documentation,  link of repository is still https://github.com/TeamHeka/medkit

**Comment by ghisvail** [comment]
Indeed, there is an ongoing rework of the documentation for the next version. It will be corrected by then.
